test/failure_handler/src/share/conf/common.properties

```diff
@@ -60,6 +62,10 @@ jstack.app=jstack
 jstack.args=-e -l %p
 jstack.params.repeat=6
 
+cores=jhsdb.jstack
+jhsdb.jstack.app=jhsdb
+jhsdb.jstack.args=jstack --mixed --core %p --exe %java
```

Comment:
strictly speaking, we can't guarantee that the executable is always `bin/java`, but it's the most common and the most interesting case, so this assumption is good enough for our pourposes.

could you please add a comment explaining this so the further reading won't be puzzled?